assets/src/app/videos/videos.component.ts

```diff
@@ -62,10 +62,10 @@ export class VideosComponent {
         this.selectedChannels = channels || this.getChannelIDsList();
         this.searchInput = params['search'] || '';
         if(params['broadcasted_after']) {
-          this.dateRange.setStartDate(moment(params['broadcasted_after']));
+          this.dateRange.setStartDate(moment(params['broadcasted_after'], "YYYY-MM-DD"));
         }
         if(params['broadcasted_before']) {
-          this.dateRange.setEndDate(moment(params['broadcasted_before']));
+          this.dateRange.setEndDate(moment(params['broadcasted_before'], "YYYY-MM-DD"));
```

Comment:
Does that set the hours:minutes:seconds to 00:00:00?

Reply:
I think Moment will do that job, because it works